### PR TITLE
jhat: add Chinese translation

### PR DESCRIPTION
**Pull request by BillLucky**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.zh/common/jhat.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# jhat
+
+> Java 堆分析工具。
+> 更多信息：<https://docs.oracle.com/javase/8/docs/technotes/tools/unix/jhat.html>.
+
+- 分析堆转储文件（来自 jmap），通过http端口 7000  进行查看：
```
Comment (marchersimon) [inline pages.zh/common/jhat.md:6]:
```suggestion
- 分析堆转储文件（来自 jmap），通过 http 端口 7000 进行查看：
```

File: pages.zh/common/jhat.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`jhat {{堆转储文件}}`
+
+- 分析堆转储文件，为 http 服务指定备用端口：
+
+`jhat -p {{端口}} {{堆转储文件}}`
+
+- 通过jhat 分析转储文件，指定使用8GB RAM（建议使用 2-4 倍的转储大小）：
```
Comment (marchersimon) [inline pages.zh/common/jhat.md:14]:
```suggestion
- 通过 jhat 分析转储文件，指定使用 8GB RAM（建议使用 2-4 倍的转储大小）：
```

**Review by marchersimon — APPROVED** [review]
Thanks @BillLucky

**Review by blueskyson — APPROVED** [review]

File: pages.zh/common/jhat.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# jhat
+
+> Java 堆分析工具。
+> 更多信息：<https://docs.oracle.com/javase/8/docs/technotes/tools/unix/jhat.html>.
+
+- 分析堆转储文件（来自 jmap），通过 http 端口 7000 进行查看：
+
+`jhat {{堆转储文件}}`
```
Comment (einverne) [inline pages.zh/common/jhat.md:8]:
```suggestion
`jhat {{路径/堆转储文件}}`
```

File: pages.zh/common/jhat.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+- 分析堆转储文件，为 http 服务指定备用端口：
+
+`jhat -p {{端口}} {{堆转储文件}}`
```
Comment (einverne) [inline pages.zh/common/jhat.md:12]:
```suggestion
`jhat -p {{端口}} {{路径/堆转储文件}}`
```

File: pages.zh/common/jhat.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+- 通过 jhat 分析转储文件，指定使用 8GB RAM（建议使用 2-4 倍的转储大小）：
+
+`jhat -J-mx8G {{堆转储文件}}`
```
Comment (einverne) [inline pages.zh/common/jhat.md:16]:
```suggestion
`jhat -J-mx8G {{路径/堆转储文件}}`
```

**Review by einverne — CHANGES_REQUESTED** [review]
Thank you for translating the page, but according to the [guideline](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md), we often add path to file when we refer a file.

By the way, I often use the `.hprof` as the file extension.

File: pages.zh/common/jhat.md (hunk 3 above)
Comment (BillLucky) [reply pages.zh/common/jhat.md:8]: OK，Thanks for your review

**Review by einverne — APPROVED** [review]

**Pull request merged**